modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/js/ClaySampleTable.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React from 'react';
+
+const ClaySampleTable = () => {
+	return (
+		<div className="container-fluid-max-lg">
+			<h3>Wrap / No wrap</h3>
+
+			<table className="table table-autofit">
+				<thead>
```

Comment:
Although this is not actually testing any table JSP taglib, I think we can consider this an exception. Said that, this sample-web module would primarily target a thorough showcase of clay taglibs, and additionally, any HTML markup making use of clay styles that we'd need to test because it belongs to our components. 

Clay react components would be outside of the scope of this module.

Do you guys agree @markocikos @kresimir-coko ?

Reply:
I know that it is a bit out of the scope as there is no taglib for the table, but as the `frontend-taglib-clay` is the module that imports the `@clayui/css` it seems to be an accurate place to put the test.
Also, the search container uses the markup and not the `<ClayTable />` component even though the test will apply to the component.